src/aliased_buffer.h

```diff
@@ -251,6 +249,12 @@ class AliasedBuffer {
   NativeT* buffer_;
   v8::Global<V8T> js_array_;
 };
+
+typedef AliasedBufferBase<int32_t, v8::Int32Array> AliasedInt32Array;
```

Comment:
Another reason for doing this is that it would be easier to implement something like `AliasedInt32Array::ForEach()` if we add a static list in the case like what we do for the `BaseObject`s - then we'll be able to iterate over all known aliased buffers in the heap snapshot generator, as well as do bookkeeping for snapshot [de]serialization (we'll record the indexes of the TypedArrays and ArrayBuffers to the snapshot during serialization, and deserialize them into C++ later to keep the states consistent)